src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.test.ts

```diff
@@ -216,7 +216,7 @@ describe('rfqEpics', () => {
         const testScheduler = new MockScheduler()
 
         testScheduler.run(({ cold, expectObservable }) => {
-          const actionLifetime = '-a 9999ms b 999ms   -|'
+          const actionLifetime = '-a 9999ms e 999ms   -|'
```

Comment:
Out on interest, does this test still pass if you'd left it a `b` ?

Reply:
no, I tried `b` before, it failed.
Then I go through this `epicTest`, and think it should change to `e`

Reply:
Great 👍